### PR TITLE
fix some include_missing bugs in declared

### PR DESCRIPTION
**Pull request by thogg4**
fixes https://github.com/ruby-grape/grape/issues/1584

### CODE REVIEW

File: spec/grape/endpoint_spec.rb [hunk 1]
```diff
@@ -450,6 +450,33 @@ def app
...
+        end
+      end
+
+      subject.get '/declared' do
+        declared(params, include_missing: false)
+      end
+
+      get '/declared?first=present&nested[fourth]=4'
+      json = JSON.parse(last_response.body)
+      p json
```
Comment (tjwp) [inline spec/grape/endpoint_spec.rb]: You probably want to remove this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 * [#1555](https://github.com/ruby-grape/grape/pull/1555): Added code coverage w/Coveralls - [@dblock](https://github.com/dblock).
 * [#1568](https://github.com/ruby-grape/grape/pull/1568): Add `proc` option to `values` validator to allow custom checks - [@jlfaber](https://github.com/jlfaber).
 * [#1575](https://github.com/ruby-grape/grape/pull/1575): Include nil values for missing nested params in declared - [@thogg4](https://github.com/thogg4).
+* [#1585](https://github.com/ruby-grape/grape/pull/1585): Fix some include_missing bugs in declared - [@thogg4](https://github.com/thogg4).
```
Comment (dblock) [inline CHANGELOG.md]: Maybe just be a tiny bit more specific please? Thx.

**Comment by dblock** [comment]
This looks good. Squash, cleanup (see comments above) and I'll merge.

**Comment by thogg4** [comment]
@dblock thanks for the review. i have made those changes.

**Comment by dblock** [comment]
Thanks, merged.

**Pull request merged**